README.md

```diff
@@ -97,6 +97,12 @@ _Testing namespace_ output. Add `:quiet true` to your `:test-refresh`
 configuration map to supporess `clojure.test`'s noisy output. This is
 particularly useful on codebases with a large number of test namespaces.
 
+### Custom Clojure.test report
```

Comment:
Could you add a space after this header? Also, could you add this feature to the list of features at the top of the README?